api/helper.ts

```diff
@@ -151,20 +151,24 @@ export function parseBtcTransactionData(
   return parsedTx;
 }
 
-export function deDuplicatePendingTx({
+export function getUniquePendingTx({
```

Comment:
Is `deDuplicatePendingTx` used anywhere else? Potentially directly imported by the app and extension?

Reply:
xverse-core doesn't export the helper.ts functions, so the export is only available locally to the package:

api/index.ts
```
export * from './btc';
export * from './xverse';
export * from './stacks';
export * from './gamma';
export * from './ordinals';
export * from './esplora/esploraAPiProvider';
export * from './ordinals/provider';
```